tailwind.config.js

```diff
@@ -1,48 +1,49 @@
 module.exports = {
   content: [
     // files that Tailwind scans for
-    'src/main/resources/templates/*.html',
-    'ts/*.ts'
+    "src/main/resources/templates/*.html",
+    "ts/*.ts",
```

Comment:
You had changed this to `ts/**/*.ts` previously, to include subdirectories, but it looks like you reversed that change. Please change it back again. This is likely why Tailwind didn't pick up one of your classes